app/controllers/api/v1/invoice_items/find_controller.rb

```diff
@@ -12,6 +12,7 @@ def index
 
         private
         def invoice_item_params
+          params[:unit_price] = (params[:unit_price].to_f*100).round if params[:unit_price]
```

Comment:
Refactor: I have a feeling this should be pulled out and placed somewhere else. Application controller? Maybe it could inherit the method so it's only written once in our app. Not quite sure where to call it from though.